mars_rover/spec/traveller_spec.rb

```diff
@@ -57,8 +57,8 @@
           it "should raise an error with an appropriate error message" do
 
               loc2 = Location.new(3,3,"E")
-              path2 = 'MMMRMMRMRRM'
-              traveller2 = Traveller.new(loc2, path2)
+              direction2 = Direction.new('MMMRMMRMRRM')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.